cinema/models.py

```diff
@@ -76,7 +76,8 @@ def __str__(self):
 class Order(models.Model):
     created_at = models.DateTimeField(auto_now_add=True)
     user = models.ForeignKey(
-        settings.AUTH_USER_MODEL, on_delete=models.CASCADE
+        settings.AUTH_USER_MODEL,
+        on_delete=models.CASCADE
```

Comment:
Ensure that `settings.AUTH_USER_MODEL` is correctly configured in your Django settings. This is important for the `Order` model to correctly reference the user model.